docs/source/deployment/nodes_grouping.md

```diff
@@ -0,0 +1,107 @@
+# Node Grouping in Kedro: Pipelines, Tags, and Namespaces
+
+When deploying a Kedro project, grouping nodes effectively is crucial for maintainability, debugging, and execution control. This document provides an overview of three key grouping methods: Pipelines, Tags, and Namespaces, along with a list of their strengths, limitations, best use cases, and links to how to use them.
+
+# Choosing between pipelines, tags, and namespaces in Kedro
+
+## Nodes grouping using pipelines in Kedro
+
+### What works with Kedro
+- If your project contains different pipelines, you can use them as predefined node groupings for deployment.
+- Pipelines can be executed separately in the deployment environment.
+- With the visualisation in Kedro Viz, you can switch to see different pipeline in a more isolated view
+<br>
+![Switching between different pipelines in Kedro Viz](../meta/images/kedro_viz_switching_pipeline.gif)
+
+### What doesn't work with Kedro
+- If you need to create custom node groupings that are different with your existing pipelines, creating new pipelines for them may not be convenient. Instead, you can use alternative grouping methods such as tags or namespaces.
+- You cannot execute more than one pipeline in a single step because the kedro run --pipeline command allows running one pipeline at a time.
```

Comment:
```suggestion
- You cannot execute more than one pipeline in a single step because the `kedro run --pipeline` command allows running only one pipeline at a time.
```

👍🏼